### PR TITLE
Fixed issue of creating df from records

### PR DESCRIPTION
**Pull request by AK-ayush**
This PR fixes the issue #914 .

### CODE REVIEW

File: python/cudf/dataframe/dataframe.py [hunk 1]
```diff
@@ -2151,12 +2151,34 @@ def from_records(self, data, index=None, columns=None, nan_as_null=False):
...
+
         df = DataFrame()
-        for k in names:
-            # FIXME: unnecessary copy
-            df[k] = Series(np.ascontiguousarray(data[k]),
-                           nan_as_null=nan_as_null)
+        if data.ndim == 2:
+            for i, k in enumerate(names):
+                # FIXME: unnecessary copy
+                df[k] = Series(np.ascontiguousarray(data[:, i]),
```
Comment (kkraus14) [inline python/cudf/dataframe/dataframe.py]: We handle making this contiguous under the hood now so no need to do this here:
```suggestion
                df[k] = Series(data[:, i],
```

File: python/cudf/dataframe/dataframe.py [hunk 2]
```diff
@@ -2151,12 +2151,34 @@ def from_records(self, data, index=None, columns=None, nan_as_null=False):
...
+                               nan_as_null=nan_as_null)
+        elif data.ndim == 1:
+            for k in names:
+                # FIXME: unnecessary copy
+                df[k] = Series(np.ascontiguousarray(data[k]),
```
Comment (kkraus14) [inline python/cudf/dataframe/dataframe.py]: Here as well:
```suggestion
                df[k] = Series(data[k],
```

File: python/cudf/dataframe/dataframe.py (hunk 1 above)
Comment (AK-ayush) [reply python/cudf/dataframe/dataframe.py]: sure

File: python/cudf/dataframe/dataframe.py [hunk 3]
```diff
@@ -2151,12 +2151,32 @@ def from_records(self, data, index=None, columns=None, nan_as_null=False):
...
+            names = columns
+
         df = DataFrame()
-        for k in names:
-            # FIXME: unnecessary copy
-            df[k] = Series(np.ascontiguousarray(data[k]),
-                           nan_as_null=nan_as_null)
+        if data.ndim == 2:
+            for i, k in enumerate(names):
+                # FIXME: unnecessary copy
```
Comment (kkraus14) [inline python/cudf/dataframe/dataframe.py]: Remove comment
```suggestion
```

File: python/cudf/dataframe/dataframe.py [hunk 4]
```diff
@@ -2151,12 +2151,32 @@ def from_records(self, data, index=None, columns=None, nan_as_null=False):
...
+                df[k] = Series(data[:, i], nan_as_null=nan_as_null)
+        elif data.ndim == 1:
+            for k in names:
+                # FIXME: unnecessary copy
```
Comment (kkraus14) [inline python/cudf/dataframe/dataframe.py]: Remove comment
```suggestion
```

File: python/cudf/dataframe/dataframe.py (hunk 3 above)
Comment (AK-ayush) [reply python/cudf/dataframe/dataframe.py]: Done

**Pull request merged**